ee/clickhouse/queries/test/__snapshots__/test_stickiness.ambr

```diff
@@ -1,75 +0,0 @@
-# name: TestClickhouseStickiness.test_aggregate_by_groups
```

Comment:
I ran a full update of snapshots - I believe these moved and were never deleted. This for example went to `test_clickhouse_stickiness`